app/models/setting.rb

```diff
@@ -175,6 +175,14 @@ def self.create!(opts)
     end
   end
 
+  def self.expand_wildcard_string(string)
+    "\\A#{Regexp.escape(string).gsub('\*', '.*')}\\Z"
+  end
```

Comment:
Can you also do better naming here? Expand does not tell what format are we expanding into.